### PR TITLE
Add hero component

### PR DESCRIPTION
**Pull request by callumacrae**
# Description

Also contains a breaking change to the breadcrumbs component - the theme schema has changed and it no longer contains vertical padding / margin (which should be the responsibility of whatever is consuming the component)

# Checklist

Pull request contains:

- [x] A new component
- [x] Component maintenance: improvement / bug fix / etc
- [ ] Component library change: storybook / webpack / etc

Definition of done:

- [x] Includes theme changes for both Uswitch and money
- [ ] Work has been tested in multiple browsers
- [x] PR description includes description of change
- [x] PR description includes screenshot of change
- [ ] If new component, designer has approved screenshot
- [x] ~If the change will affect other teams, that team knows about this change~
- [x] If introducing a new component behaviour, added a story to cover that case.
- [ ] If component change, version updated

### CODE REVIEW

File: src/elements/breadcrumbs/src/index.tsx [hunk 1]
```diff
@@ -2,24 +2,31 @@
...
+  variant === 'base' ? 'breadcrumbs2.base' : `breadcrumbs2.variants.${variant}`
+
+interface HomeIconProps extends React.HTMLAttributes<HTMLDivElement> {
+  variant?: 'base' | 'light'
+}
+
+const HomeIcon: React.FC<HomeIconProps> = ({ variant = 'base' }) => {
   const { theme }: any = useThemeUI()
 
+  const iconColor = get(theme, `${lookup(variant)}.homeIcon.color`)
```
Comment (callumacrae) [inline src/elements/breadcrumbs/src/index.tsx]: Thoughts on using `eval` or `new Function` for this? Seems pointless to load a library for what is a completely safe use of eval.

File: _element-template/src/index.tsx [hunk 2]
```diff
@@ -8,8 +8,6 @@ interface Props extends React.HTMLAttributes<HTMLDivElement> {
 }
 
 const MyComponent: React.FC<Props> = ({ someText }) => {
-  // const { theme }: any = useThemeUI()
```
Comment (jonathan-fielding) [inline _element-template/src/index.tsx]: this was left because you might want to use it in your real component

Comment (callumacrae) [reply _element-template/src/index.tsx]: It's an anti-pattern

File: src/elements/breadcrumbs/src/index.tsx (hunk 1 above)
Comment (jonathan-fielding) [reply src/elements/breadcrumbs/src/index.tsx]: write a utiltiy helper to use eval behind scenes so people knwo why we use it

**Pull request merged**